src/main/resources/Messages_fr.properties

```diff
@@ -876,6 +882,9 @@ jsp.search.facet.refine                                         = D\u00E9couvrir
 jsp.search.facet.refine.author                                  = Auteur(e)
 jsp.search.facet.refine.subject                                 = Sujet
 jsp.search.facet.refine.dateIssued                              = Date publi\u00E9
+jsp.search.facet.refine.has_content_in_original_bundle          = Contenant des fichiers
+jsp.search.facet.refine.value_has_content_in_original_bundle_true               = Oui
+jsp.search.facet.refine.value.has_content_in_original_bundle_false              = Faux
```

Comment:
I would prefer this to say `Non` (no in french) but the English options are: Yes | False

Reply:
"Yes | False" is strange in English.  I would suggest that you follow French usage for the French text.  Perhaps the English text should be corrected to "Yes | No".